source/common/upstream/BUILD

```diff
@@ -595,6 +597,15 @@ envoy_cc_library(
     alwayslink = LEGACY_ALWAYSLINK,
 )
 
+envoy_cc_library(
```

Comment:
next step to compile is add this as a dependency in lots of places around the directory tree.